src/pages/Flow/Details-Tile.vue

```diff
@@ -209,61 +297,255 @@ export default {
               </div>
             </v-list-item-content>
           </v-list-item>
+          <v-list-item dense class="px-0">
+            <v-list-item-content width="800px" class="overflow-x-scroll">
+              <v-list-item-subtitle class="caption">
+                Labels
+                <v-menu
+                  v-model="noLabelInfo"
+                  :close-on-content-click="false"
+                  offset-y
+                  open-on-hover
+                >
+                  <template v-slot:activator="{ on }">
+                    <v-btn
+                      :color="labels.length < 1 ? 'warning' : null"
```

Comment:
Same comment as below - when the color changed I thought something might be wrong or poorly configured